src/modules/market/components/market-preview-outcomes/market-preview-outcomes.jsx

```diff
@@ -1,55 +1,48 @@
 import React from 'react'
-// import ValueDenomination from 'modules/common/components/value-denomination/value-denomination';
+import PropTypes from 'prop-types'
 
 import Styles from 'modules/market/components/market-preview-outcomes/market-preview-outcomes.styles'
 
-const MarketOutcomes = p => (
-  <div className={Styles.MarketPreviewOutcomes}>
-    {/* {(p.outcomes || []).map((outcome, i) => (
-      <div
-        key={outcome.id}
-        className="outcome"
-      >
-        {!!outcome.lastPricePercent &&
-          <ValueDenomination
-            className="outcome-price"
-            {...outcome.lastPricePercent}
-            formatted={outcome.lastPricePercent.rounded}
-            formattedValue={outcome.lastPricePercent.roundedValue}
-          />
-        }
-        <span
-          data-tip
-          data-for={`outcome-name-tooltip-${outcome.marketID}-${outcome.id}`}
-          data-event="click focus"
-          className="outcome-name"
-        >
-          {outcome.name}
+const MarketOutcomes = (p) => {
+  const calculatePosition = () => {
+    const lastPrice = p.outcomes[0] && p.outcomes[0].lastPricePercent.full
```

Comment:
Use the `getValue` method found in the `utils` dir.

Example: `getValue(p.outcomes[0], ‘lastPricePercent.full’)`

This will simply return `null` if any portion of the reference is undefined, otherwise it will return the target value.